src/net_processing.cpp

```diff
@@ -74,6 +73,9 @@ static const int HISTORICAL_BLOCK_AGE = 7 * 24 * 60 * 60;
 
 // Internal stuff
 namespace {
+    /** Enable BIP61 (sending reject messages) */
+    bool enable_bip61 = DEFAULT_ENABLE_BIP61;
```

Comment:
I know the developer notes don't clarify what "global" is, but unless you have a reason to remove the prefix, I'd suggest keeping it for now to make the diff smaller?

Reply:
This commit could also be broken into 1 minor move commit and one rename scripted-diff.